### ISSUE TITLE
Uninstalling plugins makes the project-config/sync command error out

### ISSUE THREAD

**Issue by FrancisMawn** [craftcms/cms#6861]
### Description
Hi, I would like to get some clarifications around the standard procedure when removing plugins on an existing live site. 
I went on my dev server to update our Craft application with the latest version, which had some plugins removed. Then when Craft tries to apply the new projet configs, it errors out. See screenshot below:

![image](https://user-images.githubusercontent.com/45174482/93607517-6b12e600-f997-11ea-9902-a1c42fd26ff7.png)

It seems like Craft is removing the plugins before applying the config changes, which cause the update script to crash since it references stuff that no longer exists into the database?

### Steps to reproduce

1. Log onto the web server
2. run `git pull` to get the latest version (which has some plugins removed)
3. run `./craft project-config/sync` to synchronize the changes

### Additional info

- Craft version: 3.4.30
- PHP version: 7.3.17
- Database driver & version: MySQL 5.7.31
- Plugins & versions:

**Comment by brandonkelly** [comment]
Which plugin did you uninstall? Do you remember how you uninstalled it?

**Comment by FrancisMawn** [comment]
Hi, thanks for the quick response. 

These were the plugins that I've uninstalled with their version:
- "spicyweb/craft-fieldlabels": "1.2.4"
- "spicyweb/craft-neo": "2.7.25"
- "supercool/tablemaker": "2.0.1"
- "ether/seo": "3.6.6"

I first removed all the fields that were previously using these plugins.

Then from the control panel UI, under the settings/plugins section, I uninstalled the plugins and then from the same menu, removed them once they were uninstalled. 

All of this worked just fine until I went onto our dev server to try to deploy the changes.
